sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -443,6 +443,24 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val FILE_STREAM_SINK_LOG_DELETE = SQLConfigBuilder("spark.sql.sink.file.log.delete")
```

Comment:
These should be in the name space `spark.sql.streaming.*`
Also, maybe `*.sink.file*` to `fileSink`. 
@marmbrus thoughts on names?